dom_wasmjs_gopherjs.go

```diff
@@ -30,6 +30,13 @@ func RenderIntoNode(node js.Value, c Component) error {
 	return renderIntoNode("RenderIntoNode", wrapObject(node), c)
 }
 
+func toLower(s string) string {
+	// We must call the prototype method here to workaround a limitation of
+	// syscall/js in both Go and GopherJS where we cannot call the
+	// `toLowerCase` string method. See https://github.com/golang/go/issues/35917
+	return js.Global().Get("String").Get("prototype").Get("toLowerCase").Call("call", js.ValueOf(s)).String()
```

Comment:
Do you have a way of checking whether performance is impacted doing this on each `toLower` call, compared to:

```Go
func toLower(s string) string {
    return toLowerCase.Call("call", js.ValueOf(s)).String()
}

var toLowerCase = js.Global().Get("String").Get("prototype").Get("toLowerCase")
```

Also, the `string -> js.Value -> string` conversions can be short-circuited if you're willing to create a more dedicated helper for getting the node name, for example:

```Go
// +build js
func lowerNodeName(node jsObject) string {
    return toLowerCase.Call("call", node.Get("nodeName")).String()
}

// +build !js
func lowerNodeName(node jsObject) string {
    return strings.ToLower(node.Get("nodeName").String())
}
```

I don't have a sense whether the performance difference would be significant or negligible.

If performance isn't a bottneck or priority for now, this can all be considered later of course.

Reply:
I think even if performance of this is poor, it would probably be negligible AND since it only occurs on calls to `RenderBody`, `RenderInto`, `RenderIntoNode` it is not very frequently called, so as-is is probably fine for now.

Benchmarking things overall (all of Vecty) would definitely be a good idea, though that will probably come a bit later.